build/webpack/webpack.preloads.entries.json

```diff
@@ -6,5 +6,9 @@
 	"zoom": {
 		"output": "preloads/zoom",
 		"entry": "./src-built-in/preloads/zoom.js"
+	},
```

Comment:
I have a PR to make finsemble.webpack.json work for preloads too (https://github.com/ChartIQ/finsemble-seed/pull/333).